### PR TITLE
updating files

### PR DESCRIPTION
**Pull request by SamSteffen**

### CODE REVIEW

**Review by rloufoster — APPROVED** [review]

File: HousingMachineLearningModel_DRAFT1.ipynb [hunk 1]
```diff
@@ -2,7 +2,7 @@
  "cells": [
   {
    "cell_type": "code",
-   "execution_count": 1,
+   "execution_count": 6,
```
Comment (SamSteffen) [inline HousingMachineLearningModel_DRAFT1.ipynb:5]: Approved

File: HousingMachineLearningModel_DRAFT1.ipynb [hunk 2]
```diff
@@ -32,7 +32,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 3,
+   "execution_count": 7,
```
Comment (SamSteffen) [inline HousingMachineLearningModel_DRAFT1.ipynb:35]: Approved.

File: HousingMachineLearningModel_DRAFT1.ipynb [hunk 3]
```diff
@@ -41,7 +41,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 4,
+   "execution_count": 8,
```
Comment (SamSteffen) [inline HousingMachineLearningModel_DRAFT1.ipynb:44]: Approved.

File: HousingMachineLearningModel_DRAFT1.ipynb [hunk 4]
```diff
@@ -55,152 +55,21 @@
...
+      "\u001b[1;31mFileNotFoundError\u001b[0m: [Errno 2] No such file or directory: 'Resources\\\\MergedCSV_sql_FLOAT.csv'"
+     ]
```
Comment (SamSteffen) [inline HousingMachineLearningModel_DRAFT1.ipynb:72]: Approved.

File: HousingMachineLearningModel_DRAFT1.ipynb [hunk 5]
```diff
@@ -266,6 +135,27 @@
...
+      "\u001b[1;31mNameError\u001b[0m                                 Traceback (most recent call last)",
+      "Cell \u001b[1;32mIn [1], line 1\u001b[0m\n\u001b[1;32m----> 1\u001b[0m sns\u001b[38;5;241m.\u001b[39mpairplot(df)\n",
+      "\u001b[1;31mNameError\u001b[0m: name 'sns' is not defined"
+     ]
+    }
+   ],
+   "source": [
+    "sns.pairplot(df)"
+   ]
+  },
```
Comment (SamSteffen) [inline HousingMachineLearningModel_DRAFT1.ipynb:158]: Approved.

**Review by SamSteffen — COMMENTED** [review]
All changes approved.

**Comment by SamSteffen** [comment]
Finished.

**Review by rloufoster — APPROVED** [review]

**Comment by rloufoster** [comment]
Hey Sam, if these reviews are about the error in the code, it’s fine. We’re not finished. The only reason I pushed it, is so Kayla can access it from home and also because we need 4 commits by Friday. It’s just a working draft.

**Pull request closed without merging**